src/mame/casio/cz101.cpp

```diff
@@ -14,10 +28,14 @@
 #include "bus/midi/midioutport.h"
 #include "cpu/upd7810/upd7811.h"
 #include "machine/clock.h"
+#include "machine/nvram.h"
+#include "sound/upd933.h"
 #include "video/hd44780.h"
+#include "ra3.h"
```

Comment:
Please put the local project header `"ra3.h"` before the devices library headers.